ansible_collections/arista/avd/plugins/plugin_utils/cv_client/workflows/deploy_cv_pathfinder_metadata_to_cv.py

```diff
@@ -23,6 +25,39 @@
 }
 
 
+def is_pathfinder_location_supported(studio_schema: InputSchema) -> bool:
+    """Detect if pathfinder location is supported by the metadata studio"""
+    pathfinder_group_fields = get_v2(studio_schema, "fields.values.pathfinderGroup.group_props.members.values")
+    if pathfinder_group_fields is None:
+        return False
+    attributes = {"pathfinderRegion", "pathfinderAddress", "pathfinderSite"}
+    return attributes.issubset(pathfinder_group_fields)
+
+
+def is_internet_exit_zscaler_supported(studio_schema: InputSchema) -> bool:
+    """Detect if zscaler internet exit is supported by the metadata studio"""
+    return bool(get_v2(studio_schema, "fields.values.zscaler"))
+
+
+async def get_metadata_studio_schema(result: DeployToCvResult, cv_client: CVClient) -> InputSchema | None:
+    """
+    Extract and return the input schema from the metadata studio.
```

Comment:
comment can be reworded maybe 

`Download and return the input schema for the cv pathfinder metadata studio`

Reply:
fixed